src/TestableIO.System.IO.Abstractions.TestingHelpers/MockFileData.cs

```diff
@@ -34,10 +34,13 @@ public class MockFileData
         public static readonly DateTimeOffset DefaultDateTimeOffset = new DateTime(1601, 01, 01, 00, 00, 00, DateTimeKind.Utc);
 
         /// <summary>
-        /// The access control of the <see cref="MockFileData"/>.
+        /// The extensibility of the <see cref="MockFileData"/>.
         /// </summary>
+        internal FileSystemExtensibility Extensibility
+            => extensibility;
+
         [NonSerialized]
-        private FileSecurity accessControl;
+        private FileSystemExtensibility extensibility = new FileSystemExtensibility();
```

Comment:
What's the benefit of going through this abstraction  here? We could just keep the field I think?